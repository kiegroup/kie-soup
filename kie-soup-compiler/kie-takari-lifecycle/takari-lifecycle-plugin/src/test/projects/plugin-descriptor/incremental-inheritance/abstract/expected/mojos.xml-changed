<?xml version="1.0" encoding="UTF-8"?>
<mojos>
  <mojo>
    <superclasses>
      <superclasse>org.apache.maven.plugin.AbstractMojo</superclasse>
    </superclasses>
    <implementation>io.takari.lifecycle.uts.plugindescriptor.AbstractExternalMojo</implementation>
    <parameters>
      <parameter>
        <name>inheritedExternalParameter</name>
        <type>java.lang.String</type>
        <editable>true</editable>
        <description>inherited external parameter</description>
      </parameter>
    </parameters>
  </mojo>
</mojos>
