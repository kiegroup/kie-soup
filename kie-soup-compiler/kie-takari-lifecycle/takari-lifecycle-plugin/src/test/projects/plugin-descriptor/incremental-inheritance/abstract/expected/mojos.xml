<?xml version="1.0" encoding="UTF-8"?>
<mojos>
  <mojo>
    <superclasses>
      <superclasse>org.apache.maven.plugin.AbstractMojo</superclasse>
    </superclasses>
    <implementation>io.takari.lifecycle.uts.plugindescriptor.AbstractExternalMojo</implementation>
    <parameters>
      <parameter>
        <name>inheritedExternalParameter</name>
        <type>java.lang.String</type>
        <editable>true</editable>
        <description>inherited external parameter</description>
      </parameter>
    </parameters>
    <requirements>
      <requirement>
        <fieldName>inheritedExternalComponent</fieldName>
        <role>io.takari.lifecycle.uts.plugindescriptor.ComponentClass</role>
        <roleHint></roleHint>
      </requirement>
    </requirements>
  </mojo>
</mojos>
